index.html

```diff
@@ -309,7 +309,10 @@ <h1>Important Terms</h1>
 			</dd>
 			<dt><dfn data-dfn-for="element" data-export="">Hidden</dfn></dt>
 			<dd>
-				<p>Indicates that the <a>element</a> is not visible, <a>perceivable</a>, or interactive to <em>any</em> user. An element is considered <em>hidden</em> if it or any one of its ancestor elements is not rendered or is explicitly hidden.</p>
+				<p>Indicates that the <a>element</a> is absent from the accessibility tree and is not exposed to accessibility APIs. An element is considered hidden if it or any one of its ancestor elements has been explicitly hidden, either by CSS properties <code>display:none</code>, <code>visibility:hidden</code> or <code>visibility:collapse</code>, or with the aria attribue <code>aria-hidden</code>.<p>
+			<dt><dfn data-dfn-for="element" data-lt="hide from all users" data-export="">Hidden From All Users</dfn></dt>
+			<dd>
+				<p>Indicates that the <a>element</a> is not visible, <a>perceivable</a>, or interactive to <em>any</em> user. An element is considered <em>hidden from all users</em> if it or any one of its ancestor elements is not rendered or is explicitly hidden from all users. In the ARIA specifications, it is important to understand an <a>element</a> can be [=element/hidden=] by removing it from the accessibility tree but not [=element/hidden from all users=] by the use of <code>aria-hidden</code>.</p>
```

Comment:
Slight wording suggestion:

```suggestion
				<p>Indicates that the <a>element</a> is not visible, <a>perceivable</a>, or interactive to <em>any</em> user. An element is considered <em>hidden from all users</em> if it or any one of its ancestor elements is not rendered or is explicitly hidden from all users. In the ARIA specifications, it is important to understand that by using <code>aria-hidden</code> an <a>element</a> can be [=element/hidden=] but not [=element/hidden from all users=] .</p>
```

 I'm not feeling too strongly about it but had to read the long sentence twice.

Reply:
I wrote this before the comment for `hidden`; that comment might apply here, too.

Reply:
ok I did edit it!